pontoon/base/models.py

```diff
@@ -966,6 +1069,13 @@ def for_project_locale(self, project, locale, paths=None, search=None):
                 pass
             entities = entities.filter(resource__path__in=paths) or entities
 
+        if filter_search:
+            search_query = Q(**{'string__icontains': filter_search}) | Q(**{'string_plural__icontains': filter_search})
+            search_query |= Q(**{'translation__string__icontains': keyword})
```

Comment:
I'm sorry, didn't pushed updated version of the code. I'll notify you when i'm fully done. I'm doing probably too many things at once.